src/components/videocard.css

```diff
@@ -61,8 +77,29 @@
 	align-items: center;
 	padding: 0.5rem;
 	gap: 5px;
+	font-size: 0.9rem;
+	letter-spacing: 1.1;
 }
 
 .videocard-modal-action:hover {
 	background-color: var(--grey-dark-opacity);
 }
+
+.video-card:hover,
+.video-card-horizontal:hover,
+.videocard-modal-action:hover {
+	cursor: pointer;
+}
+
```

Comment:
You can make a util class for pointer

Reply:
It's on hover na, otherwise I would have made it